src/execution/operator/csv_scanner/state_machine/csv_state_machine_cache.cpp

```diff
@@ -202,6 +251,7 @@ void CSVStateMachineCache::Insert(const CSVStateMachineOptions &state_machine_op
 	transition_array.skip_standard[static_cast<uint8_t>('\n')] = false;
 	transition_array.skip_standard[static_cast<uint8_t>('\r')] = false;
 	transition_array.skip_standard[comment] = false;
+	transition_array.skip_standard[escape] = false;
```

Comment:
Should this be wrapped on the `rfc_4180` flag?

Reply:
Done